spec/services/true_layer/bank_data_import_service_spec.rb

```diff
@@ -34,7 +34,7 @@
           .to change(BankAccount, :count).by(mock_data[:accounts].count)
 
         expect(bank_provider.bank_accounts.pluck(:true_layer_id))
-          .to contain_exactly(*mock_data[:accounts].pluck(:account_id))
+          .to match_array(mock_data[:accounts].pluck(:account_id))
```

Comment:
Seems contrary to the MatchArray cop previously fixed but does not matter unless the test needs to ensure no additional array elements exist (this would pass but contains_exactly would not i believe)

Reply:
Thanks for the detail check. I think, after some digging, that it will be okay. MatchArray seems to be an exact match running the following
`> fake_array = mock_data[:accounts].pluck(:account_id)`

```
=> ["b95b007f11d9cc230d939f2bad508823", "53f74085702ee77c4ff46248446a2351"]
```

`> fake_array << "Test"`

```
=> ["b95b007f11d9cc230d939f2bad508823", "53f74085702ee77c4ff46248446a2351", "Test"]
```
`> expect(bank_provider.bank_accounts.pluck(:true_layer_id)).to match_array(fake_array)`
```RSpec::Expectations::ExpectationNotMetError:            Expected ["53f74085702ee77c4ff46248446a2351", "b95b007f11d9cc230d939f2bad508823"]
to match array with "b95b007f11d9cc230d939f2bad508823", "53f74085702ee77c4ff46248446a2351" and "Test"
```